microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -341,34 +454,51 @@
-                taken is highlighted in the main tutorial on
-                <a href="https://docs.qiime2.org/2020.6/tutorials/moving-pictures/#taxonomic-analysis" target="_blank">feature-classification</a>.</p>
-                <div style="padding:10px">
-                  <table id="taxonomyTable" class="display" style="width:100%">
-                    <thead>
-                    <tr>
-                      <th>Relative Abundance</th>
-                      <th>Kingdom</th>
-                      <th>Phylum</th>
-                      <th>Class</th>
-                      <th>Order</th>
-                      <th>Family</th>
-                      <th>Genus</th>
-                    </tr>
-                    </thead>
-                  </table>
-                </div>
+              <div class="row">
+                  <div class="column" style="text-align: center">
+                    <h3> What is in your sample? </h3>
+                    <br>
+                    <p>The table below shows the relative abundances of all of the organisms we
+                    observed in your sample. To produce this, we took the DNA sequences from
+                    your sample, and compared them against publicly available annotated
+                    reference databases. In some cases, a sequence may uniquely match a
+                    database record, in other cases a sequence may match may different records.
+                    To handle the uncertainty that can arise, we rely on a well used and
+                    publicly available microbiome software package called <a
+                    href="https://qiime2.org" target="_blank">QIIME 2</a>. The exact approach
+                    taken is highlighted in the main tutorial on
+                    <a href="https://docs.qiime2.org/2020.6/tutorials/moving-pictures/#taxonomic-analysis" target="_blank">feature-classification</a>.</p>
+                    <div style="padding:10px">
+                      <table id="taxonomyTable" class="display" style="width:100%">
+                        <thead>
+                        <tr>
+                          <th>Relative Abundance</th>
+                          <th>Kingdom</th>
+                          <th>Phylum</th>
+                          <th>Class</th>
+                          <th>Order</th>
+                          <th>Family</th>
+                          <th>Genus</th>
+                        </tr>
+                        </thead>
+                      </table>
+                    </div>
+                  </div>
+              </div>
+              <hr/>
+              <div class="row">
+                  <div class="column" style="text-align: center">
+                    <p>
+                    In the plot below, you can see the most commonly observed
+                    microbial genera in the dataset, and distribution of ranks of
+                    those genera.  For example, TOP_RANKED_GENUS_FROM_PLOT
```

Comment:
is it possible to replace `TOP_RANKED_GENUS_FROM_PLOT` with the top ranked genus?

Reply:
Yes, but relies on code from another PR, will fix once its back into st_dh_cool_results_page